music21/metadata/properties.py

```diff
@@ -0,0 +1,1732 @@
+    # and this particular sublist can be found at:
+    # https://memory.loc.gov/diglib/loc.terms/relators/dc-contributor.html
+
+    # ACT/Actor: a person or organization who principally exhibits acting
+    #   skills in a musical or dramatic presentation or entertainment.
+    PropertyDescription(
+        name='ACT',
+        namespace='marcrel',
+        uniqueName='actor',
+        valueType=Contributor,
+        isContributor=True),
+
+    # ADP/Adapter: a person or organization who 1) reworks a musical composition,
+    #   usually for a different medium, or 2) rewrites novels or stories
+    #   for motion pictures or other audiovisual medium.
+    PropertyDescription(
+        name='ADP',
+        namespace='marcrel',
+        uniqueName='adapter',
+        valueType=Contributor,
+        isContributor=True),
+
+    # ANM/Animator: a person or organization who draws the two-dimensional
+    #   figures, manipulates the three dimensional objects and/or also
+    #   programs the computer to move objects and images for the purpose
+    #   of animated film processing.
+    PropertyDescription(
+        name='ANM',
+        namespace='marcrel',
+        uniqueName='animator',
+        valueType=Contributor,
+        isContributor=True),
+
+    # ANN/Annotator: a person who writes manuscript annotations on a printed item.
+    PropertyDescription(
+        name='ANN',
+        namespace='marcrel',
+        uniqueName='annotator',
+        valueType=Contributor,
+        isContributor=True),
+
+    # ARC/Architect: a person or organization who designs structures or oversees
+    #   their construction.
+    PropertyDescription(
+        name='ARC',
+        namespace='marcrel',
+        uniqueName='architect',
+        valueType=Contributor,
+        isContributor=True),
+
+    # ARR/Arranger: a person or organization who transcribes a musical
+    #   composition, usually for a different medium from that of the original;
+    #   in an arrangement the musical substance remains essentially unchanged.
+    PropertyDescription(
+        name='ARR',
+        namespace='marcrel',
+        oldMusic21WorkId='arranger',
+        valueType=Contributor,
+        isContributor=True),
+
+    # ART/Artist: a person (e.g., a painter) or organization who conceives, and
+    #   perhaps also implements, an original graphic design or work of art, if
+    #   specific codes (e.g., [egr], [etr]) are not desired. For book illustrators,
+    #   prefer Illustrator [ill].
+    PropertyDescription(
+        name='ART',
+        namespace='marcrel',
+        uniqueName='artist',
+        valueType=Contributor,
+        isContributor=True),
+
+    # AUT/Author: a person or organization chiefly responsible for the
+    #   intellectual or artistic content of a work, usually printed text.
+    PropertyDescription(
+        name='AUT',
+        namespace='marcrel',
+        uniqueName='author',
+        valueType=Contributor,
+        isContributor=True),
```

Comment:
this could end up being confusing.

Reply:
Yeah, in music21 that would always be 'composer', surely.  I can remove it.

Reply:
(or some other more specific author)

Reply:
Removed.